### PR TITLE
Docs fix spelling issues

### PR DESCRIPTION
**Pull request by cratiu222**
Hello
I found and fixed several spelling issues in your docs.
Br, Christina.

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [aspnetcore/client-side/bundling-and-minification.md](https://github.com/dotnet/AspNetCore.Docs/blob/ec693f226437ea1cc4706f496b703377ab0fc710/aspnetcore/client-side/bundling-and-minification.md) | [Bundle and minify static assets in ASP.NET Core](https://review.learn.microsoft.com/en-us/aspnet/core/client-side/bundling-and-minification?branch=pr-en-us-33290) |
| [aspnetcore/client-side/dotnet-interop.md](https://github.com/dotnet/AspNetCore.Docs/blob/ec693f226437ea1cc4706f496b703377ab0fc710/aspnetcore/client-side/dotnet-interop.md) | [Run .NET from JavaScript](https://review.learn.microsoft.com/en-us/aspnet/core/client-side/dotnet-interop?branch=pr-en-us-33290) |

### CODE REVIEW

**Comment by cratiu222** [comment]
> @cratiu222 please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @dotnet-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > 
> > ```
> > @dotnet-policy-service agree
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> > 
> > ```
> > @dotnet-policy-service agree company="Microsoft"
> > ```
> 
> Contributor License Agreement

File: aspnetcore/diagnostics/asp0006.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ builder.CloseElement();
 
 ## How to fix violations
 
-To fix a violation of this rule, make sure that calls to methods on the <xref:Microsoft.AspNetCore.Components.Rendering.RenderTreeBuilder> class that take a sequence number as a parameter use a literal sequence number.
+To fix a violation of this rule, make sure that calls to methods on the <xref:Microsoft.AspNetCore.Components.Rendering.RenderTreeBuilder> class that takes a sequence number as a parameter uses a literal sequence number.
```
Comment (guardrex) [inline aspnetcore/diagnostics/asp0006.md:36]:
```suggestion
To fix a violation of this rule, make sure that calls to methods on the <xref:Microsoft.AspNetCore.Components.Rendering.RenderTreeBuilder> class that take a sequence number as a parameter uses a literal sequence number.
```

File: aspnetcore/diagnostics/asp0006.md [hunk 2]
```diff
@@ -33,7 +33,7 @@ builder.CloseElement();
 
 ## How to fix violations
 
-To fix a violation of this rule, make sure that calls to methods on the <xref:Microsoft.AspNetCore.Components.Rendering.RenderTreeBuilder> class that take a sequence number as a parameter use a literal sequence number.
+To fix a violation of this rule, make sure that calls to methods on the <xref:Microsoft.AspNetCore.Components.Rendering.RenderTreeBuilder> class that take a sequence number as a parameter uses a literal sequence number.
```
Comment (guardrex) [inline aspnetcore/diagnostics/asp0006.md:36]:
```suggestion
To fix a violation of this rule, make sure that calls to methods on the <xref:Microsoft.AspNetCore.Components.Rendering.RenderTreeBuilder> class that take a sequence number as a parameter use a literal sequence number.
```

**Review by guardrex — APPROVED** [review]
Thanks @cratiu222! 🚀

**Comment by guardrex** [comment]
You can't respond to the CLA with a block quote. It has to be a comment for the system to accept it. It's finicky about it 😄.

**Comment by guardrex** [comment]
@cratiu222 ... Do you want to sign the CLA? I can't merge this without it.

**Comment by guardrex** [comment]
I moved the updates over to https://github.com/dotnet/AspNetCore.Docs/pull/33291. Thanks again, @cratiu222! 🚀

**Pull request closed without merging**